Dockerfile

```diff
@@ -1,25 +1,36 @@
 FROM python:3.8
+WORKDIR /app
```

Comment:
Jeg har flyttet rundt på rækkefølgen i dockerfile så vi arbejder sammen med dens Cache. 

Samtidigt så har jeg gjort sådan at den henter SASS ned her i stedet for at den er i git.